WalletWasabi/BitcoinCore/Rpc/CachedRpcClient.cs

```diff
@@ -9,7 +9,7 @@ namespace WalletWasabi.BitcoinCore.Rpc;
 
 public class CachedRpcClient : RpcClientBase
 {
-	private CancellationTokenSource _tipChangeCancellationTokenSource;
+	private CancellationTokenSource? _tipChangeCancellationTokenSource;
```

Comment:
```suggestion
	private CancellationTokenSource _tipChangeCancellationTokenSource = new();
```
This also gets rid of the warning.
Should we really let this field be nullable? We can only get this if we ask its property and inside the `getter` we ensure even with a lock that the CTS is NOT null.

Also the CTS is never disposed, not sure why.

Reply:
Yes true, created an issue [here](https://github.com/zkSNACKs/WalletWasabi/issues/7530)!